view/frontend/web/js/view/checkout/summary/tax/mixin.js

```diff
@@ -14,8 +14,6 @@ define([], function () {
         totalTaxTitle = 'Import Fees';
```

Comment:
Hello Vlad,
could we add translation here? :)

Reply:
translation added